miasm2/expression/expression.py

```diff
@@ -399,7 +418,7 @@ def visit(self, cb, tv=None):
         return self
 
     def copy(self):
-        return ExprId(self.name, self._size)
+        return ExprId(self.name, self.size)
```

Comment:
In fact *each* attribute could be manipulated using its private version.
But don't you think it's preferable for readability to use the common name?
It could be:
```python
def copy(self):
    return ExprId(self.__name, self.__size)
```

instead of:
```python
def copy(self):
    return ExprId(self.name, self.size)
```